.gitignore

```
@@ -9,6 +9,7 @@ codecov.sh
 lcov.info
 bin/
 obj/
+**/coverage/*.json
```

changed to `cobertura` format becuase codecov requires xml

```suggestion
**/coverage/*.xml
```